documentation/staging/content/faq/boot-identity-not-valid.md

```diff
@@ -17,11 +17,22 @@ Check that the user name and password credentials stored in the Kubernetes Secre
 Then [stop all WebLogic Server instances](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers)
 in the domain before restarting so that the operator will repeat its introspection and generate the corrected `boot.properties` files.
 
-If the "Boot identity not valid" error is still not resolved, and the error is logged on a WebLogic Managed Server, then the server may be failing to contact the Administration Server. Check carefully for any errors or exceptions logged before the "Boot identity not valid" error in the Managed Server log file and look for indications of communication failure. For example:
+If the "Boot identity not valid" error is still not resolved, and the error is logged on a WebLogic Managed Server, then the server may be failing to contact the Administration Server. Check carefully for any errors or exceptions logged before the "Boot identity not valid" error in the Managed Server log file and look for indications of communication failure.
```

Comment:
"Boot identity not valid" -> `Boot identity not valid` (should be in code font, globally, as well as other error message text, such as `SSLKeyException: Hostname verification failed`)